examples/image_detection.py

```diff
@@ -31,35 +48,24 @@ def _build_ui(self):
                 ):
                     ImageDetection(
                         src="https://placecats.com/300/200",
-                        annotations=(
-                            [
-                                {
-                                    "id": 1,
-                                    "category_id": 0,
-                                    "label": "my category",
-                                    "bbox": [60, 50, 100, 100],
-                                },
-                                {
-                                    "id": 99,
-                                    "category_id": 1,
-                                    "label": "fallback label",
-                                    "bbox": [140, 100, 100, 100],
-                                },
-                            ],
-                        ),
-                        categories=(
-                            [
-                                {
-                                    "id": 1,
-                                    "name": "my category",
-                                },
-                            ],
-                        ),
+                        annotations=("annotations",),
+                        categories=("categories",),
```

Comment:
better and correct now, but I would prefer it like

```
annotations=("annotations", ANNOTATIONS),
categories=("categories", CATEGORIES),
```

and 

```
ANNOTATIONS = [...]
CATEGORIES = [...]
```

Reply:
Changed to that.